README.md

```diff
@@ -0,0 +1,53 @@
+# Filebeat
+
+Setup a filebeat instance.
+
+
+## Requirements
+
+An apt based linux system
+
+## Role Variables
+
+
+| Variable           | Default / Mandatory                                             | Description                                                                                                                                                                                                                                              |
+|--------------------|-----------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `filebeat_config`  | `{ path: { logs: /var/log/filebeat, data: /var/lib/filebeat }}` | Since the filebeat settings file is a yaml file we write the whole object to the settings file. For more Information please see the [filebeat settings file documentation](https://www.elastic.co/guide/en/filebeat/current/filebeat-settings-file.html) |
+| `filebeat_modules` | `{}`                                                            | Dictionary of config files to write to modules.d/. More information below.                                                                                                                                                                               |
+
+### `filebeat_modules`
+Each entry in the `filebeat_modules` consits out of the following entries.
+| Variable | Default / Mandatory | Description                                                                              |
+|----------|---------------------|------------------------------------------------------------------------------------------|
+| `name`   | :heavy_check_mark:  | Name of the module config file. This value has to be the same as the name of the module. |
```

Comment:
`filebeat_modules` is a dictionary according to readme. What key shall be used?

Why not use the module name as key and its configuration as the corresponding value.

Reply:
But why not use the dict for it? It worked in your other role.